web/src/api/loki.ts

```diff
@@ -22,13 +24,15 @@ export type StreamResult = {
 };
 
 export interface ParsedStream {
```

Comment:
Do you think we should rename more stuff here? I realize `ParsedStream` was probably a poor choice ...
What about `IPFIXRecord` instead? Maybe also rename `IPFIXStream` to `IPFIXBlob` or something similar?